src/core/server/ui_settings/settings/theme.ts

```diff
@@ -53,9 +54,11 @@ export const getThemeSettings = (): Record<string, UiSettingsParams> => {
       type: 'select',
       options: ['v7', 'Next (preview)'],
       description: i18n.translate('core.ui_settings.params.themeVersionText', {
-        defaultMessage: `Switch between the theme used for the current and next version of OpenSearch Dashboards, A page refresh is required for the setting to be applied.`,
+        defaultMessage: `<p>Switch between the theme used for the current and next version of OpenSearch Dashboards. A page refresh is required for the setting to be applied.</p><p><a href="{href}">{linkText}</a></p>`,
```

Comment:
```suggestion
        defaultMessage: `<p>Switch between the themes used for the current and next version of OpenSearch Dashboards. A page refresh is required for the setting to be applied.</p><p><a href="{href}">{linkText}</a></p>`,
```

Reply:
@joshuarrrr  work with documentation for copy approval.

Reply:
@joshuarrrr @AMoo-Miki The above suggested changes are correct.